services/workflows-service/src/providers/file/file.service.ts

```diff
@@ -219,6 +221,13 @@ export class FileService {
       };
     }
 
+    if (provider == 'base64' && z.string().refine(Base64.isValid).parse(uri?.split(',')[1])) {
```

Comment:
Should we use `safeParse` here or handle the error in some way? Like this the error probably won't be useful in its message and won't propagate to where we'd expect.

Reply:
Error out of it will make sure that wrong data type will be passed to the context.